x-pack/plugins/security/server/plugin.ts

```diff
@@ -209,10 +205,6 @@ export class Plugin {
         // exception may be `sessionTimeout` as other parts of the app may want to know it.
         config: {
           loginAssistanceMessage: config.loginAssistanceMessage,
-          session: {
-            idleTimeout: config.session.idleTimeout,
-            lifespan: config.session.lifespan,
-          },
```

Comment:
We only added these to `__legacyCompat` because the legacy security plugin was using them to inject default vars into the webpage. Since that is no longer happening, we don't need to pass these back to the legacy security plugin at all!